### DISCUSSION TITLE
Disable Splash Screen

### DISCUSSION THREAD

**Discussion by K80theSHADE** [rocksdanister/lively#2250] [Q&A] [answered]
Hello! I start Lively with Windows 10 startup.
Is there a setting or perhaps a commandline arg to disable the splash screen?

**Comment by rocksdanister** [comment]
Splash screen will only show up during first run or after update to do some clean up.

https://github.com/rocksdanister/lively/blob/87b494edb494a91654486d7e0973ab50557dae94/src/Lively/Lively/App.xaml.cs#L181-L184

These settings get cleared after you open the app once:
https://github.com/rocksdanister/lively/blob/87b494edb494a91654486d7e0973ab50557dae94/src/Lively/Lively.UI.WinUI/Views/MainWindow.xaml.cs#L477-L480

**Reply by K80theSHADE** [reply]
It's been doing this ever since the last update, and it runs the splash every time.

**Reply by rocksdanister** [reply, marked as answer]
> These settings get cleared after you open the app once:


Open and close the ui.

**Reply by K80theSHADE** [reply]
Cheerio! Worked! Thanks.

However, I believe the installer ought to add Lively to the System Path. This may allow the IsFirstRun parameter to update.

**Reply by rocksdanister** [reply]
There is going to be a hotfix update with multiple fixes for v2.1
Sigh can't be helped, Lively don't have any testers (beta program is a failure.) 😔
